lit_tests/kernel/wave/attention.py

```diff
@@ -1102,16 +1112,16 @@ def test_paged_flash_decoding():
         dynamic_symbols_1,
         dynamic_symbols_map_1,
     ) = get_paged_decode_attention_kernels(
-        shape, max_tokens, mfma_variant, num_kv_splits
+        shape, mfma_variant, num_kv_splits, k_shape, v_shape, block_table_shape
     )
 
     torch.manual_seed(0)
-    q = torch.randn(shape[0], shape[1], shape[3], dtype=torch.float16)
-    k = torch.randn(shape[0], shape[4], shape[3], dtype=torch.float16)
-    v = torch.randn(shape[0], shape[4], shape[2], dtype=torch.float16)
-    logits = torch.zeros(shape[0], shape[1], shape[2], dtype=torch.float32)
-    logits_max = torch.zeros(shape[0], shape[1], dtype=torch.float32)
-    output = torch.zeros(shape[0], shape[1], shape[2], dtype=torch.float32)
+    q = torch.randn(S, B, K1, dtype=torch.float16)
+    k = torch.randn(k_shape, dtype=torch.float16)
```

Comment:
It seems like you've changed the sizes and shapes of everything here? How does this relate? I was expecting the overall size to remain the same but to have things divided up into different logical dimensions, but it doesn't seem like that's the case.

Before `q` was `B x M x K1 = 128*1* 32 = 2^12 = 4096` vs now it's `S x B x K1 = 8*128*32 = 2^15 = 32768`

Before `k` was `B x K2 x K1 = 128*64*32 = 2^18 = 262,144` and now it's `S x SEQ_LEN x BH x K1 = 8*100*4*32 = 102,400` (or if we used the previous value of `SEQ_LEN = 128`, then `8*128*4*32 = 2^17 = 131,072`, half as large).

Before `v` was `B x K2 x N = 128*64*32 = 2^18 = 262,144` and now it's `S x SEQ_LEN x N x K2 = 8*100*32*64 = 1,638,400`